.mailmap

```diff
@@ -21,6 +21,8 @@ Sebastian Schuberth <sebastian.schuberth@bosch.io> <sebastian.schuberth@here.com
 Sebastian Schuberth <sebastian.schuberth@bosch.io> <sschuberth@gmail.com>
 Sebastian Schuberth <sebastian.schuberth@bosch.io> <sschuberth@users.noreply.github.com>
 Stephanie Neubauer  <Stephanie.Neubauer@bosch.io> <Stephanie.Neubauer@bosch-si.com>
+Thomas Steenbergen <thomas_steenbergen@epam.com> <thomas_steenbergen@here.com> 
```

Comment:
The HERE email address should say `<thomas.steenbergen@here.com>` (with a dot instead of an underscore).